lib/version_utils.pm

```diff
@@ -310,7 +310,7 @@ sub is_sle {
 Returns true if called on a transactional server
 =cut
 sub is_transactional {
-    return 1 if is_microos;
+    return 1 if (is_microos || is_sle_micro);
```

Comment:
since you are improving this function, would you mind adding here also the case when `TRANSACTIONAL_SERVER=1`?

```suggestion
    return 1 if (is_microos || is_sle_micro || check_var('TRANSACTIONAL_SERVER', 1));
```

Reply:
Is it something else than SYSTEM_ROLE=serverro?

Reply:
I dont know what that is, but I found an SLE test where it installs a SLE transactional mode and uses that qcow2 for the tests: https://openqa.suse.de/tests/6492110#

Reply:
https://openqa.suse.de/tests/5990679/file/vars.json yeah, system role is set to default :/
Ok, lemme add it there

Reply:
I have updated the line below this one